moveit_simple/include/moveit_simple/moveit_simple.h

```diff
@@ -388,19 +473,54 @@ class OnlineRobot : public Robot
 
 
   /**
+   * @brief plan out a given trajectory
+   * @param traj_name - name of trajectory to be executed (must be filled with
+   * prior calls to "addTrajPoint".
+   * @param goal - Joint trajectory goal
+   * @param collision_check - bool to turn check for collision on\off
+   * @throws <moveit_simple::ExecutionFailureException> (Execution failure)
+   * @throws <moveit_simple::IKFailException> (Conversion to joint trajectory failed)
+   * @throws <std::invalid_argument> (Trajectory "traj_name" not found)
+   * @throws <moveit_simple::CollisionDetected> (One of interpolated point is
+   * in Collision with itself or environment)
+   */
+  void planOnly(const std::string traj_name, 
```

Comment:
Change name to `plan`....the Only modify does not add anything.

Reply:
ok